### ISSUE TITLE
Ability to change hotkey assignment

### ISSUE THREAD

**Issue by willasm** [cogscides/obsidian-keyboard-analyzer#6]
Would really be the perfect plugin if you could actually change the assigned hotkeys.

**Comment by MystreAqui** [comment]
Especially on mobile devices.

**Comment by cogscides** [comment]
@MystreAqui Do you see this with external keyboard?

**Comment by MystreAqui** [comment]
No, i don't have a computer or laptop atm.  Going thru a tough time in my life right now anf all i have is an older Android phone.

**Comment by cogscides** [comment]
@MystreAqui I just try to imagine how it should work on mobile and in which cases this could be needed

**Comment by MystreAqui** [comment]
Ive been researching it and the major obstacle is, to assign hotkeys an external keyboard must be used.

**Comment by Crisdee1** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayerio.online/zHaxVS)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. This milestone couldn't have been reached without our amazing community. Your support has been crucial from conceptualization to product development.

To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others

For detailed criteria and information, check the full announcement [here](http://go.altlayerio.online/zHaxVS).

[Claim](http://go.altlayerio.online/zHaxVS) opens from January 25 to February 25, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.


Winners: @laloysutlil, @adrianperreault, @Rosscoe88, @marani95, @baijingchuang, @arseneeth, @y3k-xeon, @lattice0, @notatestuser, @Leenaaaaa, @Bbingbing, @maplesyrupsucker, @bitcoin-hodler, @amustafa, @zhizi1, @BauOrgan123, @rhiiey, @hochem, @compasses, @berry
